packages/grid/_modules_/grid/components/panel/filterPanel/GridFilterPanel.tsx

```diff
@@ -49,6 +50,11 @@ export function GridFilterPanel() {
     }
   }, [addNewFilter, filterModel.items.length]);
 
+  React.useEffect(() => {
+    // use an object instead of a boolean to trigger the focus of GridFilterForm by modifying the object.
+    setHasFocus({ value: true });
```

Comment:
I don't understand why to use an object. Wouldn't be better to use `filterItem.id`? In this sense, we keep which filter form to focus on. Once a filter is removed, we change to focus the next filter.

Notice that it's still jumping when opening the panel. The value of `filterModel.items.length` is 0 initially, then the first filter is added and it becomes 1.

Reply:
If I have 2 Items with ID1 and ID2 and `hasFocus='ID2'` Now, I remove the filter with ID1.

The focus disappears because it has moved to the delete button of FilterForm with ID1 which is now removed by React.

So I want to set again focus on the last FilterItem which is still ID2. Then I get `hasFocus='ID2'`  which is the same as before, and the FilterForm props `hasFocus` is still `true` so it does not trigger a next `focus()`. 

Being able to call `focus()` from the FilterPannel seems to be the good solution to handle this logic